### PR TITLE
Blog post: Interfacing podman via the varlink protocol

### PR DESCRIPTION
**Pull request by haraldh**

### CODE REVIEW

File: _posts/2018-12-13-podman-varlink.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+# Interfacing podman via the varlink protocol
+## By {{ author.display_name }} [GitHub](https://github.com/{{ author.github }}) [Twitter](https://twitter.com/{{ author.twitter }})
+
+This guide shows how to access podman remotely via the [varlink interface](https://varlink.org)
+with the CLI tools and programmatically with python and rust.
+
+Works on Linux, MacOS and Windows 10.
+
+A compatibility matrix can be seen on https://varlink.org/Language-Bindings
+
```
Comment (TomSweeneyRedHat) [inline _posts/2018-12-13-podman-varlink.md]: Coud you add a  tag here please?

File: _posts/2018-12-13-podman-varlink.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
+---
+title: Interfacing podman via the varlink protocol
```
Comment (TomSweeneyRedHat) [inline _posts/2018-12-13-podman-varlink.md]: 'podman' to 'Podman' through out please, except for actual commands.

File: _posts/2018-12-13-podman-varlink.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+
+A compatibility matrix can be seen on https://varlink.org/Language-Bindings
+
+> Note: replace 192.168.122.29 in this guide with the IP or hostname of your
+> podman machine
+
+## Set up podman on a Fedora/RHEL machine
+
+~~~bash
+$ sudo dnf install podman libvarlink-util
```
Comment (TomSweeneyRedHat) [inline _posts/2018-12-13-podman-varlink.md]: suggest 'yum' over 'dnf', it's usuable on either.

File: _posts/2018-12-13-podman-varlink.md (hunk 1 above)
Comment (haraldh) [reply _posts/2018-12-13-podman-varlink.md]: What do you mean by `tag` ?

Comment (haraldh) [reply _posts/2018-12-13-podman-varlink.md]: Something like this?
```
The [compatibility matrix](https://varlink.org/Language-Bindings) shows which feature is supported on which OS in which language.
```

Comment (TomSweeneyRedHat) [reply _posts/2018-12-13-podman-varlink.md]: Sorry, md reply obliterated the readmore tag.

```
<!--readmore-->
```

File: _posts/2018-12-13-podman-varlink.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+~~~bash
+$ ssh-keygen
+~~~
+
+Optionally, if you don't want to enter your password every time:
+~~~bash
+$ ssh-copy-id 192.168.122.29
+~~~
+
+## Python
```
Comment (TomSweeneyRedHat) [inline _posts/2018-12-13-podman-varlink.md]: When I saw ```## Python``` here, I wasn't sure if you were talking Linux or Windows here.  Might be good to have a bit more explanatory text in the ```## Windows`` section.  Feel free to chop this up, but maybe something like.  "For the next several sections this guide will demonstrate how you can run Podman commands via varlink from your Windows platform."  Or some such.

File: _posts/2018-12-13-podman-varlink.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+>>> info=podman.GetInfo()
+>>> print(info["info"]["host"]["uptime"])
+50h 23m 8.84s (Approximately 2.08 days)
+>>> print(info["info"]["host"]["os"])
+linux
+>>>
+~~~
+
+## Rust
+
```
Comment (TomSweeneyRedHat) [inline _posts/2018-12-13-podman-varlink.md]: A little explanatory text here might be good too.   Feel free to chop:  "If you'd prefer to use Rust over Python, varlink allows for that too.  The remainder of this guide will show how it's done."

**Review by TomSweeneyRedHat — COMMENTED** [review]
Looks good overall, thanks so much for putting it together!  Just a few nits and bit of explanation and I think it will be good to go.  @rhatdah WDYT?

File: _posts/2018-12-13-podman-varlink.md (hunk 4 above)
Comment (haraldh) [reply _posts/2018-12-13-podman-varlink.md]: Well, it's Linux, MacOS _and_ Windows.... 
Windows just has some more preparation steps

Comment (TomSweeneyRedHat) [reply _posts/2018-12-13-podman-varlink.md]: Yeah, I figured that out, but thought a bit of explanation on  what you were doing might help clarify for a reader.  Your call.

**Comment by haraldh** [comment]
Will continue editing tomorrow :)

**Comment by TomSweeneyRedHat** [comment]
Thanks @haraldh. it's a nice blog appreciate you putting it together.  We won't merge this tomorrow, when you do your other updates, could you rename the file to :   _posts/2018-12-17-podman-varlink.md please and we'll aim to publish it on Monday.

**Comment by TomSweeneyRedHat** [comment]
@haraldh ping?  Any progress or should we hold until after the new year?

**Comment by rhatdan** [comment]
I don't think we should publish anything until the new year.

**Comment by TomSweeneyRedHat** [comment]
reping @haraldh Can you touch this up a bit and we'll publish?

**Review by TomSweeneyRedHat — COMMENTED** [review]
LGTM Thanks @haraldh 
@baude @rhatdan I'd like to publish today, any differing thoughts?

**Comment by haraldh** [comment]
All right.. next round.. this time also with `golang` :smile:

**Comment by baude** [comment]
LGTM

**Comment by rhatdan** [comment]
@TomSweeneyRedHat I don't see this blog on podman.io?

**Comment by TomSweeneyRedHat** [comment]
I moved it.  I'd not realized Harold had dated it for next Monday.  I renamed  it.  https://podman.io/blogs/2019/01/16/podman-varlink.html and it should be showing on the main page.

**Pull request merged**